hackmd-day3.md

```diff
@@ -0,0 +1,260 @@
+    - Otherwise if you continue on the same branch, these commits get added to the open pull request which can surprise you and confuse the reviewer
+    - If your continued work depends on something that you have submitted and that is waiting, branch off from that branch and continue work. Once your first change was merged, you can rebase that new branch onto `master`.
+    - Sounds good except I don't understand the rebase part yet.
+      - Rebase can be used to "move" the starting points of a branch
+      - https://coderefinery.github.io/git-branch-design/01-rebase/
+      - It's like cutting off a branch on a tree and transplanting it to another place
+      - In this case above I would start branch2 based on branch1. Branch1 waits for code review and eventually gets merged to master. Later I can "move" (rebase) branch2 on top of master and it will look like branch2 started from master and at some point I can send a pull request from branch2.
+      - Rebasing is advanced and it modifies history and it is OK to not use it.
+      - It can help creating more linear histories.
+      - Ok, got it. Thanks!
+
+- how to add a colaborator in a project on github? Is it under setting/action permission/add runner?
+  - settings -> "manage access"
+
+
+### Day 3 feedback
+
+* Pro: The breakoutroom worked much better today. Examples are very clear and gave us good experience of working collaboratively.
+* Cons: The only thing is to keep the  text in one place to not confuse people with what the text is. Try to be consistent in time structure for class (when breaks are).
+
+* For the breakout rooms we managed to do very little, and that little was not even always correct. Next time that will have to be better prepared and at least tested, to make sure the instructions are up to date.
+
+* The examples with central->distributed were very clear, to improve: some were missunderstanding "examples" as lines that should have been executed. The remote upstream namely would benefit from different backround for "do this", and "read this". But I liked a lot.
+
+* The first part of the main session was good, although really fast and I am not sure those not familiadayr were all able understand.
+* The second part was really confusing and difficult to follow (Richard even intervened to ask what that was about) especially when the trainer went back & forth in the teaching material or did not share his terminal while typing.
+
+* A thorough recap next Tuesday at the start of the session about what we did today will be highly appreciated.
+
+- The main reason I registered for this workshop was to know more about "Collaborative distributed version control" and learn the best practices through simple exercises. I get out of here really frustrated.
+  - [name=Radovan] sorry to read this. I would love to hear what I/we can do to avoid this in future. Was it the presentation? The material? The exercises/examples? The timing? Speed?
```

Comment:
```suggestion
  - [name=Radovan] sorry to read this. I would love to hear what I/we can do to avoid this in future. Was it the presentation? The material? The exercises/examples? The timing? Speed?
  - Please contact us to let us know why.  In particular, I'm wondering - was it too fast or too slow, or a different topic than expected?  Something else?
```